memcached_stat.py

```diff
@@ -8,29 +9,24 @@
 # Global settings, one entry per instance
 CONFIGS = []
 
+class EmptyString:
```

Comment:
Okay I saw the behavior you describe by looking at the code of bmemcached, and now I fully understand why you have to do this.